tests-fuzz/src/generator/alter_expr.rs

```diff
@@ -191,6 +194,105 @@ impl<R: Rng> Generator<AlterTableExpr, R> for AlterExprModifyDataTypeGenerator<R
     }
 }
 
+/// Generates the [AlterTableOperation::SetTableOptions] of [AlterTableExpr].
+#[derive(Builder)]
+#[builder(pattern = "owned")]
+pub struct AlterExprSetTableOptionsGenerator<R: Rng> {
+    table_ctx: TableContextRef,
+    #[builder(default)]
+    _phantom: PhantomData<R>,
+}
+
+impl<R: Rng> Generator<AlterTableExpr, R> for AlterExprSetTableOptionsGenerator<R> {
+    type Error = Error;
+
+    fn generate(&self, rng: &mut R) -> Result<AlterTableExpr> {
+        let all_options = AlterTableOption::iter().collect::<Vec<_>>();
+        // Generate random distinct options
+        let mut option_templates_idx = vec![];
+        for _ in 1..rng.gen_range(2..=all_options.len()) {
+            let option = rng.gen_range(0..all_options.len());
+            if !option_templates_idx.contains(&option) {
+                option_templates_idx.push(option);
+            }
+        }
+        let options = option_templates_idx
+            .iter()
+            .map(|idx| match all_options[*idx] {
+                AlterTableOption::Ttl(_) => {
+                    let ttl: u32 = rng.gen();
+                    AlterTableOption::Ttl((ttl as i64).into())
+                }
+                AlterTableOption::TwcsTimeWindow(_) => {
+                    let time_window: u32 = rng.gen();
+                    AlterTableOption::TwcsTimeWindow((time_window as i64).into())
+                }
+                AlterTableOption::TwcsMaxOutputFileSize(_) => {
+                    let max_output_file_size: u64 = rng.gen();
+                    // Use B as the unit by default
+                    let max_output_file_size_str = max_output_file_size.to_string() + "B";
+                    AlterTableOption::TwcsMaxOutputFileSize(
+                        ReadableSize::from_str(&max_output_file_size_str).unwrap(),
```

Comment:
What about constructing the readable size directly?
```suggestion
                    AlterTableOption::TwcsMaxOutputFileSize(
                        ReadableSize(max_output_file_size),
```